tests/worksheets/src/test/scala/tests/worksheets/WorksheetSuite.scala

```diff
@@ -299,8 +299,8 @@ class WorksheetSuite extends BaseSuite {
     compat = Map(
       Compat.Scala3 ->
         """|crash:4:1: error: java.lang.RuntimeException: boom
-           |	at repl.MdocSession$App.crash(crash.scala:7)
-           |	at repl.MdocSession$App.<init>(crash.scala:15)
+           |	at repl.MdocSession$MdocSessionAppBody.crash(crash.scala:7)
```

Comment:
```suggestion
           |	at repl.MdocSession$MdocApp.crash(crash.scala:7)
```
Maybe? Will be shorter a bit.

Reply:
Nice! just renamed it to your suggestion.